src/main/java/duke/Task.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+/**
+ * This class represents each task registered by the user.
+ */
+public class Task {
```

Comment:
Good OOP! I like that you use tasks as the superclass of todo, deadline, and events to take advantage of polymorphism.